C/tests/c4PerfTest.cc

```diff
@@ -152,19 +164,113 @@ class PerfTest : public C4Test {
             b.stop();
         }
         b.printReport(1, "doc");
+        if(sf_title) {
+            string sf = generateShowfast(b, 1000.0*1000.0, sf_title);
+            writeShowFastToFile(sf_title, sf);
+        }
+    }
+    
+    
+    string generateShowfast(Benchmark &mark, double scale, string title, const char* items = nullptr) {
+        if(isEncrypted()) {
+            title += "_encrypted";
+        }
+        
+        FLEncoder enc = FLEncoder_NewWithOptions(kFLEncodeJSON, 0, false);
```

Comment:
FYI, the Fleece C++ API (Fleece.hh) is friendlier...

Reply:
I'll keep that in mind for an update!